src/lib/Libifl/int_hook.c

```diff
@@ -91,16 +90,16 @@ PBSD_hookbuf(int c, int reqtype, int seq, char *buf, int len,
 		hook_filename)) ||
 		(rc = encode_DIS_ReqExtend(c, NULL))) {
 
-		if (!rpp) {
+		if (prot == PROT_TCP) {
 			if (set_conn_errtxt(c, dis_emsg[rc]) != 0)
 				return (pbs_errno = PBSE_SYSTEM);
 		}
 		return (pbs_errno = PBSE_PROTOCOL);
 	}
 
-	if (rpp) {
+	if (prot == PROT_TPP) {
```

Comment:
If you track down rpp value all the way down, then it will be either PROT_TPP (earlier it was PROT_RPP) or PROT_TCP, and since we are trying to remove "rpp" word from code along with RPP code. I renamed rpp (and so struct batch_request -> isrpp) to prot (short for protocol) and modified all changes to explicitly check for either PROT_TPP or PROT_TCP
Because today we have 0 and 1 value of those macros but in future, we may change to something else, so whoever changes those value doesn't have to change it everywhere.